docs/userguide/apps.rst

```diff
@@ -125,10 +118,67 @@ to include attributes from the original function (e.g., its name).
     my_max_app = python_app(my_max)
```

Comment:
This is equivalent to doing:

.. code-block:: python

    @python_app
    my_max_app(*args, **kwargs):
        return my_max(*args, **kwargs)

Reply:
Very true. Nice way of tying it to something people would already think of